tests/runtime/data/stackdriver/stackdriver_test_timestamp.h

```diff
@@ -0,0 +1,62 @@
+/* 2020-07-21T16:40:42.000012345Z */
+#define TIMESTAMP_FORMAT_OBJECT_COMMON_CASE	"["		\
+	"1591111124,"			\
+	"{"				\
+        "\"timestamp\": "		\
+        "{"            \
+            "\"seconds\": \"1595349642\","          \
+            "\"nanos\": \"12345\""      \
+        "}"     \
+	"}]"
+
+/* Default: 2020-07-21T16:40:00Z */
```

Comment:
What does this default mean? Is this for `1595349600`? Please make it clear.

Reply:
Yes.
Comments updated.